starter-code/js/index.js

```diff
@@ -1,52 +1,87 @@
-function deleteItem(e){
+// function deleteItem(e){
 
-}
+// }
 
 function getPriceByProduct(itemNode){
 
-}
+  var itemQuantity = document.getElementsByTagName('input')[0]; //get user's input
+  var itemPrice = getElementsByClassName('price')[0];  //get the item's price
```

Comment:
for this one you would need to grab the info from the html tag. Since it is not an input you would use .innerHTML at the end like this . `var itemPrice = document.getElementsByClassName('price')[0].innerHTML;`